### PR TITLE
[15.0][MIG] partner_deduplicate_acl: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT36477

### CODE REVIEW

**Comment by cesar-tecnativa** [comment]
@pedrobaeza could you please check it?

**Review by pedrobaeza — APPROVED** [review]
Can you please move the top menu before "Settings" one?

**Comment by cesar-tecnativa** [comment]
> Settings

done

File: partner_deduplicate_acl/views/base_partner_merge_view.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        sequence="2"
+    />
+    <menuitem
+        id='partner_merge_automatic_menu'
+        action='base.action_partner_deduplicate'
+        groups='group_automatically'
+        parent='base_tools'
+    />
+    <record id="contacts.res_partner_menu_config" model="ir.ui.menu">
+        <field name="sequence">10</field>
```
Comment (pedrobaeza) [inline partner_deduplicate_acl/views/base_partner_merge_view.xml]: Can you avoid this overwriting?

Comment (cesar-tecnativa) [reply partner_deduplicate_acl/views/base_partner_merge_view.xml]: Ok

**Comment by cesar-tecnativa** [comment]
@LudLaf could you review this?

**Comment by pedrobaeza** [comment]
@LudLaf the commit message is not correct. It should follow commit message guidelines: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#71commit-message

**Comment by LudLaf** [comment]
Noted. I'll proceed as you've recommended.

**Comment by cesar-tecnativa** [comment]
@CarlosRoca13 please review

**Comment by pedrobaeza** [comment]
Missing proper commit message

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**